bullet-featherstone/src/Base.hh

```diff
@@ -551,6 +606,10 @@ class Base : public Implements3d<FeatureList<Feature>>
       if (link->collider)
       {
         world->world->removeCollisionObject(link->collider.get());
+        GzCollisionDispatcher *dispatcher =
+            dynamic_cast<GzCollisionDispatcher *>(
+            world->world->getDispatcher());
+        dispatcher->RemoveManifoldByCollisionObject(link->collider.get());
```

Comment:
Should we remove the object from the `colPairManifolds` map too? Otherwise, that map will grow indefinitely, which might become a problem if objects are added and removed frequently.

Reply:
yes removed them from `colPairManifolds`. [15d4ede](https://github.com/gazebosim/gz-physics/pull/664/commits/15d4ede77239f9a1b35971a13a9c3fe4ccaa1bca)